### PR TITLE
test updated repository-to-cpe.json format

### PR DESCRIPTION
**Pull request by RTann**
https://access.redhat.com/security/data/metrics/repository-to-cpe.json added a new field to the JSON. This PR simply ensures current live Scanners will be able to continue parsing the file correctly

### CODE REVIEW

**Review by daynewlee — APPROVED** [review]

File: pkg/repo2cpe/testdata/repository-to-cpe.json [hunk 1]
```diff
@@ -2,23 +2,37 @@
...
         "cpe:/a:redhat:3scale_amp:2.8::el8",
         "cpe:/a:redhat:3scale_amp:2.9::el8"
+      ],
+      "repo_relative_urls": [
+        "content/dist/layered/rhel8/x86_64/3scale-amp/2/debug"
       ]
     },
     "3scale-amp-2-rpms-for-rhel-8-x86_64-rpms": {
       "cpes": [
+        "cpe:/a:redhat:3scale:2.13::el8",
```
Comment (jvdm) [inline pkg/repo2cpe/testdata/repository-to-cpe.json:18]: @RTann Why did you add these additional CPEs to test the new `repo_relative_urls` field?

Comment (RTann) [reply pkg/repo2cpe/testdata/repository-to-cpe.json:18]: I just copied it directly from the current version of https://access.redhat.com/security/data/metrics/repository-to-cpe.json. Yes, it does not affect the test and added some noise, but I figured I might as well just copy over the entire current state of these entries

Comment (jvdm) [reply pkg/repo2cpe/testdata/repository-to-cpe.json:18]: :+1:
